file_manager/run_file.py

```diff
@@ -0,0 +1,96 @@
+import threading
+import os
+import traceback
+
+import pexpect
+
+from helpers.logger import create_logger
+
+
+FILE_PATH = '/home/username/Code_Docs/running_files/'
+DOCKER_IMAGES = {'python': 'code_docs_python'}
+
+
+run_file_logger = create_logger("run_file_logger")
+
+
+class RunFileThread(threading.Thread):
+    def __init__(self, file_name, file_content, programming_language, consumer=1):
+        super().__init__(name="run file")
+        self.filename = self.create_file(file_name, file_content)
```

Comment:
what happens if start two files with some filename?

Reply:
This shouldn't happen